android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -124,6 +133,50 @@ class JsonConverter {
             return tweaksConfig
         }
 
+        /**
+         * Converts any JS object into an `AdvertisingConfig` object.
+         * @param json JS object representing the `AdvertisingConfig`.
+         * @return The generated `AdvertisingConfig` if successful, `null` otherwise.
+         */
+        @JvmStatic
+        fun toAdvertisingConfig(json: ReadableMap?): AdvertisingConfig? = json?.getArray("schedule")
+            ?.toList<ReadableMap>()
+            ?.mapNotNull(::toAdItem)
+            ?.let { AdvertisingConfig(it) }
```

Comment:
That's some goodness to read ❤️  Already paid off to introduce the extension function 💪